docker/Dockerfile

```diff
@@ -53,6 +53,14 @@ RUN set -e; \
 
 RUN set -e; \
     export DEB_MAX_ATTEMPT=10; \
+    export CRIS_IMG_ARCH=$(case $(uname -m) in \
+    amd64 | x86_64 | x64 | *) \
```

Comment:
Where can we get x64? Thought that's Microsoft-only terminology.

Reply:
I don't where I can get it. I guess that is the problem. Are there any documentations on the possible output of `uname -m`? If not then it is somehow still a hack here.

Reply:
No, I think it's just a string and people can write whatever they want.
So probably based on what you have seen?
If it's me I would probably also google for uname output of popular devices in doubt.

Reply:
> I think it's just a string and people can write whatever they want.

That is what I think too. So imo we just put whatever we have seen or imagined here.